Makefile

```diff
@@ -423,3 +423,23 @@ generate-flag-testdata:
 
 install_kubectl_kind:
 	./tools/get_kubectl_kind.sh
+
+
+
+# Scripts related to test coverage
+PACKAGE ?= ./go/...
+
+coverage:
+	go install golang.org/x/tools/cmd/cover
+	go test -coverprofile=coverage.out $(PACKAGE)
+	go tool cover -html=coverage.out
```

Comment:
This feels very similar to the already existing `unit_test_cover` Makefile target. The following snippet is going to be important is testing the `sqlparser` package:
```sh

# Handle go tool cover failures due to not handling `//line` directives, which
# the goyacc compiler adds to the generated parser in sql.go. See:
# https://github.com/golang/go/issues/41222
sed -i'' -e '/^vitess.io\/vitess\/go\/vt\/sqlparser\/yaccpar/d' coverage.out
```

Reply:
Ahh got it. Should I rename the same to `test.coverage` to avoid the target conflict?